_posts/2023-02-23-aries-robotics-sponsorship.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Our New Sponsor: Aries Robotics"
+---
+
+<img src="{{ '/images/sponsors/aries-robotics.svg' | prepend: site.baseurl }}" />
```

Comment:
The logo feels a bit on the big side -- I'd be tempted to add some padding around it?

Reply:
[4fe4804](https://github.com/srobo/website/pull/490/commits/4fe4804e844478cb4b544e58cfae1f6de9437d9f)